src/app/global-vars.service.ts

```diff
@@ -717,28 +730,63 @@ export class GlobalVarsService {
     this.amplitude.logEvent(event, data);
   }
 
-  launchLoginFlow() {
-    this.logEvent("account : login : launch");
-    this.identityService.launch("/log-in").subscribe((res) => {
-      this.logEvent("account : login : success");
-      this.backendApi.setIdentityServiceUsers(res.users, res.publicKeyAdded);
-      this.updateEverything().subscribe(() => {
-        this.flowRedirect(res.signedUp);
-      });
+  openJumio(jumioSuccessRoute: string, jumioErrorRoute: string): void {
+    // Note: this endpoint will fail if success and error routes do not conform to the expectations of Jumio.
+    // See here for details: https://github.com/Jumio/implementation-guides/blob/master/netverify/netverify-web-v4.md#url-requirements
+    this.backendApi
+      .JumioBegin(
+        environment.jumioEndpointHostname,
+        this.loggedInUser?.PublicKeyBase58Check,
+        jumioSuccessRoute,
+        jumioErrorRoute
+      )
+      .subscribe(
+        (res) => {
+          window.open(res.URL);
```

Comment:
I'd recommend copying `identityService.launch` and specifying the size and location of the new window you're opening.